src/Numerics/Spline2/BsplineAllocator.hpp

```diff
@@ -67,7 +68,7 @@ class BsplineAllocator
    * @param coeff array of coefficients
    * @param spline target MultibsplineType
    */
-  void setCoefficientsForOneOrbital(int i, Array<T, 3>& coeff, SplineType* spline);
+  void setCoefficientsForOneOrbital(int first, int last, Array<T, 3>& coeff, SplineType* spline);
```

Comment:
The function name should probably change, since it appears that more than one orbital is being initialized.   Maybe 'setCoefficientsforOrbitals'?

Reply:
Good point. Fixed the function name.
1.1138 sec to 0.04 sec for a 375MB table. This is GNU on x86